aspnetcore/fundamentals/url-rewriting/samples/2.x/SampleApp/README.md

```diff
@@ -1,8 +1,8 @@
 # ASP.NET Core URL Rewriting Sample (ASP.NET Core 2.x)
 
-This sample illustrates usage of ASP.NET Core 2.x URL Rewriting Middleware. The application demonstrates URL redirect and URL rewriting options. For the ASP.NET Core 1.x sample, see [ASP.NET Core URL Rewriting Sample (ASP.NET Core 1.x)](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/url-rewriting/samples/1.x).
+This sample illustrates usage of ASP.NET Core 2.x URL Rewriting Middleware. The app demonstrates URL redirect and URL rewriting options.
 
-When running the sample, a response will be served that shows the rewritten or redirected URL when one of the rules is applied to a request URL.
+When running the sample, a response is served that shows the rewritten or redirected URL when one of the rules is applied to a request URL, except with the redirecting an XML file request and rewriting a text file request examples. For the file examples, Static File Middleware serves the file after the request URL is rewritten by the middleware.
```

Comment:
>"is applied to a request URL, except with the redirecting an XML ..."

nit: This sentence feels very long. I'd break this up into something like
is applied to a request URL. This though doesn't apply to examples redirecting an XML ..." or something like that.